packages/uniforms-material/src/NumField.tsx

```diff
@@ -28,11 +28,13 @@ function Num({
 }: NumFieldProps) {
   return (
     <TextField
-      disabled={!!disabled}
+      disabled={disabled}
       error={!!error}
+      fullWidth={props.fullWidth ?? true}
```

Comment:
I'd rather use `fullWidth = true,` in the arguments lists as we do that in other components.